.drone.yml

```diff
@@ -97,6 +97,6 @@ pipeline:
     <<: *helm_deploy
     prefix: PROD
     secrets: [ PROD_API_SERVER, PROD_KUBERNETES_TOKEN ]
-    values: ingress.globalStaticIpName=code-annotation-production,ingress.hostname=code-annotation.sourced.tech,image.tag=${DRONE_TAG},deployment.gaTrackingID=UA-109494282-2,secretName=code-annotation-drone-workaround
+    values: ingress.globalStaticIpName=code-annotation-production,ingress.hostname=code-annotation.srcd.run,image.tag=${DRONE_TAG},deployment.gaTrackingID=UA-109494282-2,secretName=code-annotation-drone-workaround
```

Comment:
is this conf compatible @rporres ?
```
ingress.globalStaticIpName=code-annotation-production
ingress.hostname=code-annotation.srcd.run
```
I'm not sure if we should use a new `globalStaticIpName` suffixed as `-production` while we're having a `hostname` following the `staging` pattern.

Is this something I should care about?
Does it affect the place where you store the secrets and these configurations?
(most probably I'm mixing concepts, so please ping me when you have time to explain it to me irl :dancer: )

Reply:
Please use `code-annotation-production.srcd.run` as hostname. The rest is ok

Reply:
But that's the hostname defined by @mcuadros in the GH OAuth App as he wrote at https://github.com/src-d/code-annotation/issues/217#issuecomment-378615285

Should we ask him to change it?

Reply:
d'oh, ok. I don't like that name, but I guess it's late to change it. I will change it on my side...